### DISCUSSION TITLE
Youtube interface not loading

### DISCUSSION THREAD

**Discussion by 7thripr** [Alex313031/Thorium-Win#66] [Q&A]
I just installed thorium browser but my youtube interface is not loading, videos are playing but the remaining interface is not .
![image](https://github.com/Alex313031/Thorium-Win/assets/67912473/0a4e9273-c14c-414e-abb4-870e7c69abf4)

Please help 
Thanks

**Comment by gz83** [comment]
The local test can open youtube normally and load all page elements, please make sure your network can access youtube normally, and make sure there are no extensions or experimental flags to prevent the loading of youtube pages.

@7thripr

**Reply by 7thripr** [reply]
I have specifically set al the experimental extensions to disabled instead of default. I started having problem after i logged in to my google account and only shows the stuff on recommended page when YT sidebar is enabled.

**Reply by 7thripr** [reply]
*Correction happens even when logged out

**Reply by gz83** [reply]
Use the same configuration as thorium in the original chrome/chromium, and test again to see if you will encounter problems

@7thripr
